src/operator/src/error.rs

```diff
@@ -198,6 +214,28 @@ pub enum Error {
         source: catalog::error::Error,
     },
 
+    #[snafu(display("Failed to find view info for: {}", view_name))]
+    FindViewInfo {
+        view_name: String,
+        #[snafu(implicit)]
+        location: Location,
+        source: common_meta::error::Error,
+    },
+
+    #[snafu(display("View info not found: {}", view_name))]
+    ViewInfoNotFound {
+        view_name: String,
+        #[snafu(implicit)]
+        location: Location,
+    },
```

Comment:
These two errors looks identical...

Reply:
No, `FindViewInfo` has the source `source: common_meta::error::Error`. It's the error of calling `ViewInfoMananager.get` which returns an `Option<ViewInfo>`. `ViewInfoNotFound` is for this option value.